Provide automation (both CloudFormation and Terraform) to create the IAM resources for an AWS Cloud Account (#1350)
RedisLabs/redislabs-docs: content/rc/how-to/creating-iam-resources/cloudformation.md

---

> Title: CloudFormation

Reviewer: ```suggestion
Title: Create resources (CloudFormation)
linkTitle: CloudFormation
```

Weight: 10
categories: ["RC"]
alwaysopen: false
---

> Simply clicking on the link below will launch the relevant CloudFormation stack creation process in the AWS console:

Reviewer: ```suggestion
The following link uses CloudFormation to create a stack using the AWS console:
```

Reviewer: It would be nice to add an opening sentence describing setting the stage for folks who land on this page from search results, rather than navigating in with the TOC.  (It can be brief.)
